### PR TITLE
add Java update/install guide

### PR DESCRIPTION
**Pull request by sulu5890**
Instructions for shared hosts have been removed as no longer relevant. As Mojang now uses Java 17 and it is not only Paper on 11+ pretty much every host supports 17 out of the box.

### CODE REVIEW

File: docs/common/java-install.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+YUM is used on older releases of CentOS/RHEL, and excessively old releases of Fedora.
+
+```bash
+sudo rpm --import https://yum.corretto.aws/corretto.key
+sudo curl -Lo /etc/yum.repos.d/corretto.repo https://yum.corretto.aws/corretto.repo
+sudo yum -y install java-17-amazon-corretto-devel
+```
+
+## Windows 10 & 11
```
Comment (MiniDigger) [inline docs/common/java-install.md:91]: maybe mention winget here, the windows package manager (nobody really cares about I guess but its the fastest way):
`winget install -e --id Amazon.Corretto.17`

File: docs/common/java-install.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+"Startup" tab of your server, select `ghcr.io/pterodactyl/yolks:java_17` from the "Docker Image"
+dropdown as shown in the image below.
+
+![Pterodactyl Manual Java Version Change](/img/docs/guides/java/pterodactyl-manual.png)
+
+The Verifying Installation section does not apply for Pterodactyl.
+
+## Verifying Installation
+
+Now that you have installed Java 17, run this command to ensure the process was successful.
```
Comment (MiniDigger) [inline docs/common/java-install.md:138]: maybe mention stuff like restarting terminal session here

File: docs/common/java-install.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+`openjdk 17` and contains `64-Bit` in the last line.
+
+```
+openjdk 17 2021-09-14 LTS
+OpenJDK Runtime Environment Corretto-17.0.0.35.1 (build 17+35-LTS)
+OpenJDK 64-Bit Server VM Corretto-17.0.0.35.1 (build 17+35-LTS, mixed mode, sharing)
+```
+
+If your installation has failed, do not hesitate to reach out on our
+[Discord](https://discord.gg/papermc) for support.
```
Comment (MiniDigger) [inline docs/common/java-install.md:154]: maybe mention the #paper-help channel explicitly here?

File: docs/common/java-install.md (hunk 1 above)
Comment (sulu5890) [reply docs/common/java-install.md:91]: Is this included out of the box? The docs Microsoft provides are a bit unclear as they provide installation instructions, which say that it is included in a default package, but then show you how to install it through the MS store.

On my windows computer, it wasn't, although admittedly it is through work so it's possible their image excludes it/provides some super outdated version not including it.

Comment (MiniDigger) [reply docs/common/java-install.md:91]: uuuhm, I don't remember installing it but this install is really old sooooo

**Comment by sulu5890** [comment]
ok, I'm happy enough with this

**Pull request merged**